### ISSUE TITLE
Feature Request: Allow adding mowers by serial-number without account

### ISSUE THREAD

**Issue by fightforlife** [Sdahl1234/Sunseeker-lawn-mower#12]
I think it woukd be cool to allow adding mowers by entering the serial-number without email and password.
This way there is no problem when using an Oauth-provider.
Also you can track known movers without having them in your personal account. (e.g. monitoring the mower of my parents)

I will try to come up with a PR, but I do not now python, so it will take some time.
Maybe someone is quicker.

My idea would be:
- Ask for Account or SN
- if Account is entered normal flow with login, token, devicelist etc....
- if serial is entered, jump over the login, token, devicelist and directly subscribe to the mqtt broker topics.

Best regards!

**Comment by garret** [comment]
@fightforlife have you followed up on your wish? Mowing season is coming back again :)

**Comment by fightforlife** [comment]
Sadly no, I tried to create a new fork of the addon, which only needs the SN. But without much knowledge about the plugin architecture I hit too many blockers for me.
What I did in the end was to use nodered to connect and subscribe to mqtt. And then convert/forward the needed values to HA.

**Comment by Sdahl1234** [comment]
You need more information than just the info from the mqtt server. That's why you need to login with your account.

**Comment by Sdahl1234** [comment]
In my opinion, it is a bad idea to allow access to a mower with only the serial number.
The app has poor security and I don't want to make it possible to hack a mower that does not belong to you.
Why is it that you can't just create an account with email and password and the integration will import the mowers you have added.

I hope you can understand it from my perspective. 😄

**Comment by garret** [comment]
@fightforlife I am not a programmer but does it mean that you have found a way to control the robot lanwmower fully local or you still need to use the sunseeker cloud service to control it?

**Comment by fightforlife** [comment]
No, sadly not fully local, it is still using the public sunseeker mqtt server.
But here someone described a way to use it completely locally.
https://github.com/OlliKantola/Sunseeker_LawnMower_Control
 You need to setup your own mqtt server and create a NAT rule in your router which redirects the mqtt traffic.
